src/SurfaceFluxes.jl

```diff
@@ -37,13 +42,11 @@ include("UniversalFunctions.jl")
 import .UniversalFunctions
 const UF = UniversalFunctions
 
-abstract type SurfaceFluxesModel end
+abstract type SolverScheme end
+struct FVScheme <: SolverScheme end
```

Comment:
It would be good to have an option for `FDScheme` here, as that is what we have right now (i.e., we do not need the FV corrections).

Reply:
this option was added (basically the GDScheme renamed)